### ISSUE TITLE
[guides] Dark mode support fails when scrolling on wide screens

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1820]
Initially, both user guide and QUick guide respect dark mode well, however in full screen the right side fails to continue to load dark mode as the screen is scrolled lower down into the guides.

The jaring experience of the light mode at the same time as the dark mode can be problematic to some users.
THis only begins to fail when the browser width is greater than 1600px.  more narrow widths and zooming will avoid this issue.  

#### Because many/some people who use dark mode:
-  who are developers without vision disabilities, this may not be a problem for them
-  who also have vision disabilities often use zooming, so they may not experience this issue
-  who are neurodivergent with no vision disability, **will** also experience this issue 

Step to reproduce:
1. Set OS to dark mode
2. Open Checker
3. Insure browser is set to width greater than 1600px
4. Open both Quick guide and User guide
5. Scroll down the guide

Screenshots showing issue

### Quick guide
![QuickGuideDarksplit](https://github.com/IBMa/equal-access/assets/16193609/d0fb09c6-7fb6-45a2-a3fd-382b7b24a63e)

### User guide
![UserGuidesDarksplit](https://github.com/IBMa/equal-access/assets/16193609/a56a1463-1091-4dc7-b1d7-292a4863df4d)

**Comment by marcjohlic** [comment]
@drjoho this is something you can take a look at during any free time during this Release.  Don't prioritize this over your other Release items.

**Comment by drjoho** [comment]
Fixed:
<img width="1913" alt="image" src="https://github.com/IBMa/equal-access/assets/64442976/9976cca1-aaab-4ea1-b3ff-f71cb228b12b">

**Comment by drjoho** [comment]
<img width="1913" alt="image" src="https://github.com/IBMa/equal-access/assets/64442976/d528006b-76db-4da4-9613-ff0df97474fa">

**Comment by philljenkins** [comment]
Well, its only fixed in the User guide, the issue is still in the Quick guide

![Screenshot 2024-02-06 at 12 01 33 PM](https://github.com/IBMa/equal-access/assets/16193609/0125eb2b-1a80-4d64-b2eb-2acacbba2683)

**Comment by drjoho** [comment]
Ok will fix dark mode for Quick guide too. All controls are disabled until scan is complete.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Phill Jenkins ***@***.***>
Sent: Tuesday, February 6, 2024 12:19:47 PM
To: IBMa/equal-access ***@***.***>
Cc: Josiah Hoskins ***@***.***>; Assign ***@***.***>
Subject: [EXTERNAL] Re: [IBMa/equal-access] [guides] Dark mode support fails when scrolling on wide screens (Issue #1820)

I noticed that the Focus VIew controls also have contrast problems prior to running the scan. Is that on purpose to signify that the control is inactive until a scan is completed? Prior to scan Screenshot. 2024-02-06. at. 12. 09. 52. PM. png (view
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
<https://us-phishalarm-ewt.proofpoint.com/EWT/v1/PjiDSg!2U-u7PHVaVMyko-QWytv5Ns1j-XBWaWFrmp0vfHs2dysAU8XxX591hwgZX2nWffQHAKdp0Cji--gonUAFeZBr8mFjS8szMc_oGeqMzW9LA$>
Report Suspicious

ZjQcmQRYFpfptBannerEnd

I noticed that the Focus VIew controls also have contrast problems prior to running the scan. Is that on purpose to signify that the control is inactive until a scan is completed?

Prior to scan

Screenshot.2024-02-06.at.12.09.52.PM.png (view on web)<https://github.com/IBMa/equal-access/assets/16193609/04806c7a-c461-42fb-8163-f6e888ce3a24>

AFTER SCAN

Screenshot.2024-02-06.at.12.14.47.PM.png (view on web)<https://github.com/IBMa/equal-access/assets/16193609/bf3e99d6-db24-46cb-beee-55437350a40f>

**Comment by drjoho** [comment]
@Phill I will look at what you have found.

**Comment by drjoho** [comment]
Quick guide filxed.

Here is what I see before scan:

<img width="1211" alt="image" src="https://github.com/IBMa/equal-access/assets/64442976/2ec2562c-ca8e-436c-8e4b-e172b227f74f">

**Comment by philljenkins** [comment]
@drjoho 
JAWS reads the controls as "unavailable", which is correct. 
Not using the `tab` key to navigate, but using the `right arrow keys` to reads the UI:
"group start Focus view",
"less than HTML, tab, unavailable", 
"All, tab, unavailable, selected"
"group end Focus view"
"Keyboard Checker Mode button unavailable" 
"Violation button collapsed"
"Need review button collapsed"
etc.

**Comment by ErickRenteria** [comment]
Merged
